src/misc/setup.cpp

```diff
@@ -1789,3 +1790,35 @@ const char* SetProp(std::vector<std::string>& pvars)
 
 	return return_msg;
 }
+
+void Config::ParseArguments()
+{
+	arguments.printconf = cmdline->FindBoolArgument("printconf", true);
+	arguments.noprimaryconf = cmdline->FindBoolArgument("noprimaryconf", true);
+	arguments.nolocalconf = cmdline->FindBoolArgument("nolocalconf", true);
+	arguments.fullscreen  = cmdline->FindBoolArgument("fullscreen", true);
+	arguments.list_glshaders = cmdline->FindBoolArgument("list-glshaders", true);
+	arguments.noconsole   = cmdline->FindBoolArgument("noconsole", true);
+	arguments.startmapper = cmdline->FindBoolArgument("startmapper", true);
+	arguments.exit        = cmdline->FindBoolArgument("exit", true);
+
+	arguments.eraseconf = cmdline->FindBoolArgument("eraseconf", true) ||
+	                      cmdline->FindBoolArgument("resetconf", true);
+	arguments.erasemapper = cmdline->FindBoolArgument("erasemapper", true) ||
+	                        cmdline->FindBoolArgument("resetmapper", true);
+
+	arguments.version = cmdline->FindBoolArgument("version", true, 'v');
+	arguments.help    = cmdline->FindBoolArgument("help", true, 'h');
```

Comment:
As I mentioned above, `FindRemoveBoolArgument("help", 'h')` etc. would be much nicer. Second best would be:

```cpp
constexpr auto remove_arg = true;
...
FindBoolArgument("help", remove_arg, 'h');
...
```

Reply:
I did that 2nd approach just to avoid duplicated code.  I made `FindBoolArgument` a private method because it was only used in once place inside the `CommandLine` class.